PulseSampleApp/PulseSampleApp/Info.plist

```diff
@@ -26,6 +26,11 @@
 	<string>LaunchScreen</string>
 	<key>UIMainStoryboardFile</key>
 	<string>Main</string>
+	<key>NSAppTransportSecurity</key>
```

Comment:
Additionally, in real world usage, third party ads can be (and are) booked as http, or will redirect to other third party networks which will serve over http, etc, so there is no solid way around this.